src/redux/reducers/addUser.js

```diff
@@ -0,0 +1,14 @@
+import { ADD_USER } from "../actions/addUserAction";
+
+export default function addUser(state = {}, action) {
+  switch (action.type) {
+    case ADD_USER:
+      console.log("Name: " + action.payload.name);
+      console.log("Surname: " + action.payload.surname);
+      console.log("Email: " + action.payload.email);
+      console.log("Test Result: " + action.payload.testResult);
+      return { ...action.payload };
```

Comment:
What you achieve here is that whenever an action of type ADD_USER takes place, the store's addUser field gets updated with an object containing the data received in payload (name, surname, email, testResult).

If you want to access this data in some way (to show the last added participant, for example - since it's not actually a user you're supposed to be adding :wink: ), you'd need to map store (using stateToProps function) to props in a way where you access this object (or its specific fields).